src/Pos/Client/Txp/Util.hs

```diff
@@ -225,6 +226,22 @@ createMTx utxo hwdSigners outputs =
         HM.lookup (AddressIA addr) signers
 
 -- | Make a multi-transaction using given secret key and info for
+-- outputs. Don't override TxDistr
+createTxNoOverride ::
```

Comment:
What's the purpose of this function? It can create invalid transactions. Do I understand correctly that the only purpose is to test that invalid transaction is rejected? If yes, it should be documented.